src/Goodby/CSV/Import/Standard/Lexer.php

```diff
@@ -50,9 +50,14 @@ public function parse($filename, InterpreterInterface $interpreter)
         }
 
         $csv = new SplFileObject($url);
-        $csv->setCsvControl($delimiter, $enclosure, $escape);
         $csv->setFlags($flags);
 
+        if (empty($enclosure)) {
+            $csv->setCsvControl($delimiter);
+        } else {
```

Comment:
__[phpmd]__
* The method parse uses an else expression. Else is never necessary and you can simplify the code to work without else.